spec/unit/target/build_settings/aggregate_target_settings_spec.rb

```diff
@@ -713,7 +713,7 @@ def specs
               @target.pod_targets.each { |pt| pt.spec_consumers.each { |sc| sc.stubs(:frameworks => %w(UIKit), :libraries => %w(z c++)) } }
 
               @xcconfig = @generator.generate
-              @xcconfig.to_hash['OTHER_LDFLAGS'].should == '$(inherited) -ObjC -l"c++" -l"z" -framework "UIKit"'
+              @xcconfig.to_hash['OTHER_LDFLAGS'].should == '$(inherited) -ObjC -no_compact_unwind -l"c++" -l"z" -framework "UIKit"'
```

Comment:
where is this flag coming from? it's unclear how this shows that the issue is fixed, though I haven't looked at the fixtures for this

Reply:
From the tests:
```ruby        
before do
          @target_definition = fixture_target_definition
          @specs = specs
          @specs.first.user_target_xcconfig = { 'OTHER_LDFLAGS' => '-no_compact_unwind', 'USE_HEADERMAP' => 'NO' } unless @specs.empty?
          @specs.first.pod_target_xcconfig = { 'CLANG_CXX_LANGUAGE_STANDARD' => 'c++11' } unless @specs.empty?
          @pod_targets = @specs.map { |spec| pod_target(spec, @target_definition) }
          @target = fixture_aggregate_target(@pod_targets, false, { 'Release' => :release }, [], Platform.new(:ios, '6.0'), @target_definition)
          unless @specs.empty?
            @target.target_definition.whitelist_pod_for_configuration(@specs.first.name, 'Release')
          end
          @generator = AggregateTargetSettings.new(@target, 'Release')
        end
```

Reply:
it'd be great if you could add a more explicit test, so a future cleanup doesn't accidentally regress behavior again

Reply:
Will do.